lib/index.js

```diff
@@ -185,33 +188,43 @@ MapboxGeocoder.prototype = {
     this._eventEmitter.emit('loading', { query: searchInput });
 
     // Possible config proprerties to pass to client
-    var keys = ['bbox', 'limit', 'countries', 'types', 'language', 'reverseMode'];
+    var keys = [
+      'bbox',
+      'limit',
+      'countries',
+      'types',
+      'language',
+      'reverseMode'
+    ];
     var self = this;
     // Create config object
     var config = keys.reduce(function(config, key) {
       if (self.options[key]) {
         // countries, types, and language need to be passed in as arrays to client
         // https://github.com/mapbox/mapbox-sdk-js/blob/master/services/geocoding.js#L38-L47
-        ['countries', 'types', 'language'].indexOf(key) > -1 ?
-          config[key] = [self.options[key]]
-        : config[key] = self.options[key]
+        ['countries', 'types', 'language'].indexOf(key) > -1
+          ? (config[key] = self.options[key].split(', '))
```

Comment:
Would this close #120 by allowing spaces?

In mapbox-gl-geocoder v2.3.0 you could pass a comma separated string without spaces, for backwards compatibility I think we should continue to support this.

Reply:
Yes, would close #120. Fixed in latest commit.